internal/provider/incident_workflow_resource.go

```diff
@@ -562,7 +563,7 @@ func toPayloadSteps(steps []IncidentWorkflowStep) []client.StepConfigPayload {
 	for _, step := range steps {
 		out = append(out, client.StepConfigPayload{
 			ForEach:       step.ForEach.ValueStringPointer(),
-			Id:            step.ID.ValueStringPointer(),
```

Comment:
I take it we made these non-computed or equivalent?

Reply:
Yeah exactly - this change is because I re-genned the client which now has them as required to match this